lib/ansible/modules/web_infrastructure/ansible_tower/tower_inventory_source.py

```diff
@@ -268,7 +285,7 @@ def main():
                 source_script_res = tower_cli.get_resource('inventory_script')
                 try:
                     script = source_script_res.get(
-                        name=module.params.get('source_script'))
+                        name=module.params.get('source_script'), organization=org['id'])
```

Comment:
This restricts items like the source script and the source project to be under the same organization as the inventory. But it is possible to use resources from different organizations as well.